### ISSUE TITLE
Can't download attachment from Web Money Manager Ex

### ISSUE THREAD

**Issue by mrcsmcl** [moneymanagerex/moneymanagerex#3341]
MMEX version: 1.5.1
Operating System: Windows

# Description of the bug

A few months ago there was an issue here about error in download attachment, but there wasn't no solution. Finally I found the reason for the errors and this is due to the double slash (\\) being inserted when saving the file, for this reason it was giving error writting on the disk.

![image](https://user-images.githubusercontent.com/21992269/116413023-9bfca880-a80d-11eb-9521-a177b8de8246.png)

![image](https://user-images.githubusercontent.com/21992269/116413062-a8810100-a80d-11eb-8b0f-973de606508b.png)

![image](https://user-images.githubusercontent.com/21992269/116413193-c64e6600-a80d-11eb-9e12-0494fd662c69.png)

While in the Attachment Setting an slash is already added.

![image](https://user-images.githubusercontent.com/21992269/116413333-e41bcb00-a80d-11eb-9dee-5f05d32394e0.png)

# Reproduction

Is the bug reproducible? 
 - [X] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

1. Add and attachment on web manager and try to download from desktop app.
…

Expected result:

The attachment be downloaded and inserted to the database with the transaction.

Actual result:

Error.
![image](https://user-images.githubusercontent.com/21992269/116413023-9bfca880-a80d-11eb-9521-a177b8de8246.png)

**Comment by vomikan** [comment]
Try to execute:
Win+R 
```c:\\\\temp\\\\```
It would be Ok for OS. 
It's probably something else. Try running the app as an administrator.

**Comment by whalley** [comment]
Closing as duplicate of https://github.com/moneymanagerex/moneymanagerex/issues/2639
